src/app/core/market/market.service.ts

```diff
@@ -41,7 +41,7 @@ export class MarketService {
     return this._http.post(this.url, postData, { headers: headers })
         .map((response: any) => response.result)
         .catch((error: any) => {
-
+          // Inconsistency when you do add to cart
```

Comment:
More explanation?

Reply:
we are getting 404 error while try to add same item in the cart and got error something like `error.error.error`, Due to which we got `[object object]` in the error message.

Reply:
Then fix the error handling..

Reply:
The in-code comment here is useless.

Reply:
ok